tools/topology/topology2/include/common/tokens.conf

```diff
@@ -162,6 +162,7 @@ Object.Base.VendorToken {
 		in_interleaving_style	1906
 		in_fmt_cfg		1907
 		in_sample_type		1908
+		pin_index		1909
```

Comment:
does this belong in this PR? I don't see the link with input/output

Reply:
It's a token needed for the pin_index attributes in the new classes